src/components/GrampsjsPerson.js

```diff
@@ -52,22 +52,8 @@ export class GrampsjsPerson extends GrampsjsObject {
     if (!this.data.profile) {
       return ''
     }
-    const surname = this.data.profile.name_surname || '…'
-    const suffix = this.data.profile.name_suffix || ''
-    const call = this.data?.primary_name?.call
-    let given = this.data.profile.name_given || call || '…'
-    const callIndex = call && call !== given ? given.search(call) : -1
-    given =
-      callIndex > -1
-        ? html`
-            ${given.substring(0, callIndex)}
-            <span class="given-name"
-              >${given.substring(callIndex, callIndex + call.length)}</span
-            >
-            ${given.substring(callIndex + call.length)}
-          `
-        : given
-    return html`${given} ${surname} ${suffix}`
```

Comment:
Again, please keep the existing version as a fallback for backwards compatibility.

By the way, does the Gramps default option for `name_display` show the suffix? If not, I am concerned users will complain about issue #284 being back.